app/views/api/services/settings.html.slim

```diff
@@ -3,4 +3,4 @@
 = semantic_form_for @service, url: admin_service_path(@service) do |form|
   = render partial: 'api/services/forms/integration_settings', locals: { form: form, service: @service }
   = form.actions do
-    = form.commit_button 'Update Product', button_html: { name: 'update_settings' }
+    = form.commit_button 'Update Product', button_html: { name: 'update_settings', value: 'true' }
```

Comment:
Looks good but what does this do?

Reply:
It ensures that when the error occurs, it doesn't redirect to the edit template, as you can see here: 

https://github.com/3scale/porta/blob/0018732a81e13aec4dfb3ab871dc7f0bcfcbd4d7/app/controllers/api/services_controller.rb#L72

Reply:
So `value: 'true'` is just to make `#present?` return true in case there are no fields? Could you explain a little bit?

Reply:
Yes, this is another thing related to the Formtastic upgrade. At the previous version, that commit button was returning `true` by default, but after the upgrade we have to add the `value: 'true'` so the user stays at `:settings`, which was part of the issue, the user being redirected to `:edit` when sending the form with an invalid URL.